test/metabase/query_processor/middleware/cache_test.clj

```diff
@@ -248,11 +248,10 @@
       (mt/wait-for-result save-chan)
       (is (= :cached (run-query))))))
 
-#_
 (deftest min-ttl-test
   (testing "if the cache takes less than the min TTL to execute, it shouldn't be cached"
     (with-mock-cache [save-chan]
-      (mt/with-temporary-setting-values [query-caching-min-ttl 60]
+      (mt/with-temporary-setting-values [query-caching-min-ttl 1000]
         (run-query)
         (is (= :metabase.test.util.async/timed-out
                (mt/wait-for-result save-chan)))
```

Comment:
If I understand correctly, you can also pass a lower wait time to `(mt/wait-for-result)`. the default is 200ms

Reply:
It's not having any effect anyways, since all of this is done sequentially. So I probably can just to speed up the test. :)